templates/support_channels.html

```diff
@@ -6,295 +6,46 @@
 {% brand_institution_name as institution %}
 
 <section class="frontpage frontpage-section" aria-labelledby="students-title">
```

Comment:
`aria-labelledby="students-title"` is now incorrect since you removed the heading with that id. Accessibility requires that sections are labeled. If the page has no suitable visible title, then the attribute `aria-label` can define the accessible title directly.

It would also be better to use different `<section>` elements for the teacher and student sections. The contents are clearly intended for different audiences. Then, the accessible section titles can highlight the student and teacher sections.